bapt/__init__.py

```diff
@@ -112,3 +113,83 @@ def read_config(filename):
                 compound['vb_gradient'] = gradients[ids[0]]
                 compound['cb_gradient'] = gradients[ids[1]]
     return band_edge_data, settings
+
+
+def get_plot_novac(data, height=5, width=None, emin=None, emax=None, colours=None,
+                   bar_width=3, show_axis=False, hide_cbo=False, hide_vbo=False,
+                   label_size=15, plt=None, gap=0.5, font=None, show_ea=False,
+                   name_colour='w', fade_cb=False):
+
+    width = (bar_width/2. + gap/2.) * len(data) if not width else width
+    emin = emin if emin else min([d['vbo'] for d in data]) - 1
+    emax = emax if emax else max([d['vbo'] + d['eg'] for d in data]) + 1
+
+    plt = pretty_plot(width=width, height=height, plt=plt, fonts=[font])
+    ax = plt.gca()
+
+    pad = -0.4
+    for i, compound in enumerate(data):
+        x = i * (bar_width + gap)
+        ip = compound['vbo']
+        ea = compound['vbo'] + compound['eg']
+
+        fade = 'fade' in compound and compound['fade']
+        edge_c_cb = '#808080' if fade or fade_cb else 'k'
+        edge_c_vb = '#808080' if fade else 'k'
+        edge_z = 4 if fade else 5
+
+        vc = vb_cmap if 'vb_gradient' not in compound else \
+            compound['vb_gradient']
+        cc = cb_cmap if 'cb_gradient' not in compound else \
+            compound['cb_gradient']
+
+        gbar(ax, x, ip, bottom=emin, bar_width=bar_width, show_edge=True,
+             gradient=vc, edge_colour=edge_c_vb, edge_zorder=edge_z)
+        gbar(ax, x, ea, bottom=emax, bar_width=bar_width, show_edge=True,
+             gradient=cc, edge_colour=edge_c_cb, edge_zorder=edge_z)
+
+        dashed_arrow(ax, x + bar_width/6., ip - pad/3, 0,
+                     ea-ip + 2 * pad/3, colour='k', line_width=_linewidth)
+        ax.text(x + bar_width/4., ip + pad/2 + compound['eg']/2,
+                '{:.2f} eV'.format(compound['eg']), ha='left',
+                va='bottom', size=label_size, color='k', zorder=2)
+
+        ax.text(x + bar_width/2., ip + pad/2, compound['name'], zorder=2,
+                ha='center', va='top', size=label_size, color=name_colour)
+        if i > 0:
+            if not hide_vbo:
+                ax.text(x + bar_width/2., ip + 1.65*pad,
+                        f"VBO: {compound['vbo'] - data[i-1]['vbo']:+.2f} eV", zorder=2,
+                        ha='center', va='top', size=label_size-3, color=name_colour,
+                        fontfamily='serif')
```

Comment:
Can you keep this the default font to be consistent with the rest of the style of the plot.

Reply:
Yep no problem. Thought it looked slightly nicer with a different font at the time, but I see that a consistent style is better 😃